src/App.js

```diff
@@ -276,6 +277,9 @@ function Root() {
     async function initializeApplication() {
       await initSentry(); // must be set up immediately
 
+      // must happen immediately, but after Sentry
+      await migrate();
```

Comment:
can we tweak the name on this fn so its more obv its running migrations, feels a lil off rn

Reply:
Any ideas? Had this thought too, but migrations might be more than just MMKV, so ended up with generic name. In the file itself the functions are `runMigrations` (multiple) and `runMigration` (single).

Reply:
We can totally change this later if we want, but imma merge for now 👍